rosa_getting_started_sts/rosa-sts-deleting-access-cluster.adoc

```diff
@@ -1,12 +1,18 @@
 include::modules/attributes-openshift-dedicated.adoc[]
 [id="rosa-sts-deleting-access-cluster"]
-= Deleting access to a ROSA cluster
+= Revoking access to a ROSA cluster
 :context: rosa-sts-deleting-access-cluster
 
 toc::[]
 
-Delete access to a {product-title} (ROSA) cluster using the `rosa` command-line.
+Access to a {product-title} (ROSA) cluster is controlled through the identity provider (IDP) accounts of users. To remove access of a user to a cluster, you must configure that within the Identity Provider (IDP) account that was set up for authentication.
```

Comment:
Can we sharpen this sentence to be written as "Access to a(ROSA) cluster is controlled through **an** identity provider (IDP). To remove access of a user to a cluster, you must configure that within the Identity Provider (IDP)  that was set up for authentication."?

An IdP may not be an account so I recommend removing references to an idp account.

Reply:
Thanks, incorporated this feedback.